duktape.go

```diff
@@ -1,17 +1,24 @@
 package duktape
 
 /*
-#cgo !windows CFLAGS: -std=c99 -O3 -Wall -fomit-frame-pointer -fstrict-aliasing
-#cgo windows CFLAGS: -O3 -Wall -fomit-frame-pointer -fstrict-aliasing
+#cgo !windows CFLAGS: -std=c99 -O3 -fomit-frame-pointer -fstrict-aliasing
+#cgo windows CFLAGS: -O3 -fomit-frame-pointer -fstrict-aliasing
 #cgo linux LDFLAGS: -lm
 #cgo freebsd LDFLAGS: -lm
 #cgo openbsd LDFLAGS: -lm
-
+#cgo CFLAGS: -I${SRCDIR}/dukluv/lib/uv
+#cgo CFLAGS: -I${SRCDIR}/dukluv/lib/uv/include
+#cgo LDFLAGS: ${SRCDIR}/dukluv/build/libdschema.a
```

Comment:
@WorkForPizza - I couldn't figure out how to compile dukluv dynamically so cross-compilation might be a pain. Perhaps we could compile all of these assets once for each goos/goarch and then use those?